{
	"title": "CVE-2019-15486",
	"cve": "CVE-2019-15486",
	"notes": [
		{
			"type": "Description",
			"text": "django-js-reverse (aka Django JS Reverse) before 0.9.1 has XSS via js_reverse_inline."
		},
		{
			"title": "Published",
			"type": "Other",
			"text": "2019-08-23"
		},
		{
			"title": "Modified",
			"type": "Other",
			"text": "2019-08-23"
		}
	],
	"references": [
		{
			"url": "https://github.com/ierror/django-js-reverse/compare/v0.9.0...v0.9.1",
			"description": "MISC:https://github.com/ierror/django-js-reverse/compare/v0.9.0...v0.9.1"
		},
		{
			"url": "https://github.com/ierror/django-js-reverse/pull/81",
			"description": "MISC:https://github.com/ierror/django-js-reverse/pull/81"
		}
	]
}
